### PR TITLE
Added a local version of the extension method Platform.IsMono.

### LINKED ISSUE #10: Fails with SMAPI 3.9.5 because of missing method EnvironmentUtility.IsMono
SMAPI has removed some "unused" methods in the 3.9.5 release. `EnvironmentUtility.IsMono` was one of them, which, unfortunately, is in use here. Luckily the method is pretty trivial and can be easily added as a local extension method.

### PR DESCRIPTION
**Pull request by tamas-kozma**
The method was recently (3.9.5) removed from SMAPI. This should fix #10.

### CODE REVIEW

File: StardewXnbHack/Framework/PlatformExtensions.cs
Comment (Pathoschild) [inline StardewXnbHack/Framework/PlatformExtensions.cs]: We can simplify this a bit:
```suggestion
            return platform == Platform.Linux || platform == Platform.Mac;
```

File: StardewXnbHack/Framework/PlatformExtensions.cs
Comment (Pathoschild) [inline StardewXnbHack/Framework/PlatformExtensions.cs:6]: Missing code docs:
```suggestion
    /// <summary>Provides extension methods for the <see cref="Platform" /> enum.</summary>
    internal static class PlatformExtensions
```

**Review by Pathoschild — CHANGES_REQUESTED** [review]
Thanks! I just have two suggested changes, but I can do those after merging if needed.

**Comment by tamas-kozma** [comment]
Thank you for reviewing my changes. I feel so dumb for just copying the code from the original repository and not realizing that Platform was simply an enum. Sorry about that and thanks for the suggested changes.
I'm not used to working with GitHub and pull requests. I've clicked a Commit button for both suggestions. Was that all I had to do?

**Comment by Pathoschild** [comment]
Yep!

**Review by Pathoschild — APPROVED** [review]

**Comment by Pathoschild** [comment]
Merged into `develop` for the upcoming StardewXnbHack 1.0.4. Thanks for the fix!

**Pull request merged**
